demos/src/js/chartAuthoring-demo-overview.js

```diff
@@ -41,7 +57,10 @@ var demo = demo || {};
             storeModel: {
                 type: "fluid.modelComponent",
                 options: {
-                    gradeNames: ["fluid.prefs.cookieStore"],
+                    gradeNames: [
+                        "fluid.prefs.cookieStore",
+                        "fluid.prefs.cookieStore.writable"
+                    ],
                     model: {
                         expander: {
                             funcName: "{that}.get"
```

Comment:
The cookie store is now a dataSource. DataSource [get](https://github.com/fluid-project/infusion/blob/a79da1b9c996e6546896d468e8f30ef7aa1325ea/src/framework/core/js/DataSource.js#L177-L182) and [set](https://github.com/fluid-project/infusion/blob/a79da1b9c996e6546896d468e8f30ef7aa1325ea/src/framework/core/js/DataSource.js#L194-L207) methods return promises. These promises are resolved with the values. This expander is expecting the stored data to be returned.